libpoolprotocols/PoolManager.cpp

```diff
@@ -437,8 +442,20 @@ void PoolManager::rotateConnect()
                          to_string(m_Settings.connections.at(m_activeConnectionIdx)->Port());
         p_client->setConnection(m_Settings.connections.at(m_activeConnectionIdx));
         cnote << "Selected pool " << m_selectedHost;
-
-        p_client->connect();
+ 
+        
+        if ((m_connectionAttempt > 1) && (m_Settings.delayBeforeRetry > 0))
+        {
+            // sleep before retry 
+            cnote << "Connect attempt " << m_connectionAttempt << ", sleep "<< m_Settings.delayBeforeRetry << "s before re-connect";
```

Comment:
Better wording.
I suggest 
`Next connection attempt in xx seconds`

Reply:
fixed in https://github.com/ethereum-mining/ethminer/pull/1824/commits/eb95ad3ff0c00561f54b881424141c031a677da2